openscenario/openscenario_utility/setup.cfg

```diff
@@ -0,0 +1,4 @@
+[develop]
+script-dir=$base/lib/openscenario_utility
+[install]
+install-scripts=$base/lib/openscenario_utility
```

Comment:
Replace the word separator hyphens in `script-dir` and `install-scripts` with underscores to suppress the warning.

```
/usr/local/lib/python3.8/dist-packages/setuptools/dist.py:697: UserWarning: Usage of dash-separated 'script-dir' will not be supported in future versions. Please use the underscore name 'script_dir' instead
```